spec/services/preserved_object_handler_spec.rb

```diff
@@ -106,4 +106,26 @@
       po_handler.create_after_validation
     end
   end
+
+  describe 'endpoint validation' do
+    it 'errors when endpoint is not an Endpoint object' do
+      poh = described_class.new(druid, incoming_version, incoming_size, 1)
+      poh.valid?
+      expect(poh.errors.messages).to include(endpoint: ["must be an actual Endpoint"])
+    end
+    it 'errors when endpoint_type is not online' do
+      endpoint_type = EndpointType.create(type_name: 'foo', endpoint_class: 'archive')
+      archival_endpoint = Endpoint.create(endpoint_name: 'archive1',
+                                          endpoint_type_id: endpoint_type.id,
+                                          endpoint_node: 'node',
+                                          storage_location: 'somewhere')
+      poh = described_class.new(druid, incoming_version, incoming_size, archival_endpoint)
+      poh.valid?
```

Comment:
For these, how about:
`expect(poh).not_to be_valid` ?

That calls `.valid?` **and** makes sure it is returning the correct value.  Since we will be checking `.valid?` (or `.invalid?`) before we ever go looking at `.errors`, it seems reasonable to have expectations on it.